features/FxA-89-devices-view/README.md

```diff
@@ -411,6 +409,17 @@ Acceptance criteria:
       by accessing the devices view on a desktop browser,
       and then go to the sync preferences on the mobile device,
       I immediately see that the device is disconnected.
+* [ ] - Moved from Phase 2: When viewing my device list
```

Comment:
One consequence of moving this to Phase 3 is that, if you don't have any devices connected, your device view looks like this with just the "get app" placeholders:

![screen shot 2016-11-18 at 19 00 25](https://cloud.githubusercontent.com/assets/34695/20422908/526e5cc2-adc1-11e6-809f-21b72e122578.png)

Are we comfortable with that @ryanfeeley @davismtl, or would you like us to prioritize some work on getting a "Connect..." button in there?